index.html

```diff
@@ -5871,7 +5881,38 @@ <h3><dfn>Get Page Source</dfn></h3>
 </section> <!-- /Get Page Source -->
 
 <section>
-<h3>Executing Script</h3>
+ <h3>Scripts</h3>
+
+ <p>
+  An <dfn>executable script</dfn> is an abstraction used to
+  identify a script when it is transported via the
+  <a href="#protocol">protocol</a>, between <a>remote</a> and
+  <a>local</a> ends.
+
+  Each script may include an optional associated <dfn>executable script name</dfn>
+  that uniquely identifies the script within the <a>current session</a>.
+
+  Each <a>session</a> maintains an <a>ordered map</a> <dfn>executable script map</dfn>,
+  with each entry having a key of an <a>executable script name</a>
+  and a value of an <a>exeuctable script</a>.
+  This map is initially empty.
+
+  To add an <a>executable script</a> to the session,
+  for <a>executable script map</a>, key <a>executable script name</a>, and value <var>script</var>,
+  set map[key] to value.
```

Comment:
This is not a normative definition because it is never called anywhere. We only ever add something to this map when under the _Pin Script_ command, so it would be sufficient to make this entire paragraph a step under this command.